extensions/wikia/TemplateClassification/scripts/TemplateClassificationModal.js

```diff
@@ -193,6 +193,7 @@ function ($, w, mw, loader, nirvana, tracker, labeling) {
 	function updateEntryPointLabel(templateType) {
 		$typeLabel
 			.data('type', mw.html.escape(templateType))
+			.children().first()
```

Comment:
If I'm reading this right, .children() is actually a lot faster: http://aahacreative.com/2010/07/19/jquery-fastest-method-find-descendents/